bin/cli.js

```diff
@@ -73,13 +73,32 @@ program.on('--help', function() {
   console.log('');
 });
 
-// Make serve the default command
 var args = process.argv;
-if (!args[2] || !program.commands.some(c => c.name() === args[2])) {
+if (!args[2]) {
```

Comment:
What happens if I just do `parcel build`? Won't this display a warning message specific to the `serve` command?